tools/docker/daml-all-in-one/healthcheck.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+# Load the environment variables from the .env file
+# if [ -f .env ]; then
+#   export $(cat .env | grep -v '#' | awk '/=/ {print $1}')
+#   echo "possible"
+# fi
+jwt_content=$(cat jwt)
```

Comment:
@raynatopedrajeta If this is no longer needed for anything then please just remove the commented code.